### DISCUSSION TITLE
Apps stuck at startup

### DISCUSSION THREAD

**Discussion by teamunicycle** [flet-dev/flet#194] [Q&A]
On a Windows 10 PC with the default view and the web browser view, even the most basic Hello World hangs, with just a loading animation. It does work on another PC I have, so I'm sure there is something about my PC, but I'm not sure how to go about troubleshooting.
Can anyone suggest a cause, or steps I might take?

**Comment by FeodorFitsner** [comment]
Run this program with log enabled and let me know what you got:

```python
import logging

import flet
from flet import Page, Text

logging.basicConfig(level=logging.DEBUG)

def main(page: Page):

    page.title = "MyApp"
    page.add(Text("Hello!"))

flet.app(target=main, view=flet.FLET_APP)
```

**Reply by teamunicycle** [reply]
Thanks Feodor. After adding logging, the apps worked. I took it out, and they still worked. I've just been pasting in your response, Hello World, and the Counter example. Now it is working every time. 
I might sit on this for a while, and see if I can catch it with the logging enabled. I'll update this thread if I do.

**Comment by TobiasHT5** [comment]
I'm facing the same issue. I just pasted code for the example of data tables but changed view to ft.WEB_BROWSER and it's stuck on the loading flet screen. If the app is run without the browser, it works fine.

::edit::
I've discovered the issue. Running flet in the browser requires an internet connection. I was offline by the time I tested. Getting back online, the app worked.
